mllib/src/main/scala/org/apache/spark/ml/classification/OneVsRest.scala

```diff
@@ -294,6 +296,18 @@ final class OneVsRest @Since("1.4.0") (
   @Since("1.5.0")
   def setPredictionCol(value: String): this.type = set(predictionCol, value)
 
+  /** @group expertGetParam */
+  override def getParallelism: Int = $(parallelism)
+
+  /**
+   * @group expertSetParam
+   * The implementation of parallel one vs. rest runs the classification for
```

Comment:
Also, please put the group annotation at the bottom to match existing code style.